### PR TITLE
OCPQE-26408 filter out microshift advisory

### PR DESCRIPTION
**Pull request by rioliu-rh**
details please refer to https://issues.redhat.com/browse/OCPQE-26408

### CODE REVIEW

**Review by tomasdavidorg — APPROVED** [review]
/lgtm just a small comment.

File: oar/core/advisory.py [hunk 1]
```diff
@@ -75,7 +77,7 @@ def change_ad_owners(self):
                 # if the advisory is released, the state is like [REP_PREP/SHIPPED LIVE], it is not [QE], we should not send alert to ART
                 # only check if the state is [NEW_FILES]
                 # talked with ART, microshift advisory should be excluded from this check
```
[the remark this answers is not in the document]
Comment (rioliu-rh) [reply oar/core/advisory.py]: done, can you check again

**Pull request merged**